index.html

```diff
@@ -3941,16 +3941,28 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
 </pre>
         
           <p>If present, the <span class="chunk">cLLi</span> chunk identifies two characteristics:</p>
+
+          <p>The <span class="chunk">cLLi</span> chunk adds static metadata which provides an opportunity
+          to optimize tone mapping of the associated content to a specific target display. This is 
+          accomplished by tailoring the tone mapping of the content itself to the specific peak brightness 
+          capabilities of the target display to prevent clipping. The method of tone-mapping optimization 
+          is currently subjective.</p>
              
-          <p>MaxFALL (Maximum Frame Average Light Level) indicates the maximum value of the <a>frame</a> 
-             average light level (in cd/m<sup>2</sup>, also known as nits) of the entire playback sequence. MaxFALL is 
-             calculated by first averaging the decoded luminance values of all the pixels in each frame,
-             and then using the value for the frame with the highest value.</p>
+          <p>MaxFALL (Maximum Frame Average Light Level) uses a static metadataa value to indicate the 
+            maximum value of the <a>frame</a> average light level (in cd/m<sup>2</sup>, also known as nits) 
+            of the entire playback sequence. MaxFALL is calculated by first averaging the decoded luminance 
+            values of all the pixels in each frame, and then using the value for the frame with the highest value.</p>
              
-          <p>MaxCLL (Maximum Content Light Level) indicates the maximum light level of any single 
-             pixel (in cd/m<sup>2</sup>, also known as nits) of the entire playback sequence.  There is often an algorithmic 
-             filter to eliminate false values occurring from processing or noise that could adversely 
-             affect intended downstream tone mapping.</p>
+          <p>MaxCLL (Maximum Content Light Level) uses a static metadata value to indicate the maximum light 
+            level of any single pixel (in cd/m<sup>2</sup>, also known as nits) of the entire playback sequence. 
+            There is often an algorithmic filter to eliminate false values occurring from processing or noise 
+            that could adversely affect intended downstream tone mapping.</p>
+
+          <p class="note"><a href="https://shop.cta.tech/products/hdr-static-metadata-extensions">CTA 861.G</a> is available 
```

Comment:
It doesn't say that much, and makes no mention of filtering to remove outliers. A much better reference would be, as noted in the issue, 
https://ieeexplore.ieee.org/document/9508136

Reply:
Please update the PR to use this reference.